### ISSUE TITLE
Combat Busts

### ISSUE THREAD

**Issue by Jacques00** [OXOIndustries/Trials-in-Tainted-Space#625]
* Busts that are formatted for group fights (using the boundaries) don't always seem to show for some reason. It is possible that I might be setting them up wrong, but the images usually show on the first group fight from game load, but with any group battle after that fight, the enemy portraits don't show at all. This issue carries on to loading a save file as well, so the game needs to be reloaded to reset the issue.

**Comment by Gedan** [comment]
Point me at a specific example to look at- I know I did some weird group-level staging for the busts, wherein theres basically three "sets" that can get displayed:

Everybody involved
Only the hostile group
Only the friendly group

The latter two there should show up on the groups respective "I have done a thing" turns. Some of the turn "layering" is probably not very visible yet, because how exactly things are broken up into multiple screen clears depends on the specific group sizes and what creature types are involved.

**Comment by Jacques00** [comment]
From what I tested so far, Saendra's X-pak1 with the assassin group fight displays on the first game load. If I did a fight before that one, it will display the fourth or first bust, or none at all. All of the KaraQuest2 busts don't seem to show at all, probably because I keep starting off in the sewers where the drones have no bust image; however, the grunts and other enemy encounters don't display ther busts at all. I haven't gotten to test the other group fights, but I assume the same issues apply.
